src/test/java/com/github/fge/jsonpatch/serialization/ExtendedJsonPatchOperationSerializationTest.java

```diff
@@ -11,7 +13,7 @@
     protected ExtendedJsonPatchOperationSerializationTest(final JsonPatchOperationFactory operationFactory)
         throws IOException
     {
-        super("extended", operationFactory);
+        super("extended", operationFactory, new JsonPatchDeserializer(JsonPatchFactoryUtil.extendedFactory()));
```

Comment:
I have to think about it, but I'm confused as to the use of both operationFactory and extendedFactory